src/nodes/NodeConnectionManager.ts

```diff
@@ -407,12 +416,18 @@ class NodeConnectionManager {
     port = 0 as Port,
     reuseAddr = false,
     ipv6Only = false,
+    enableMdns = true,
+    mdnsGroups = ['224.0.0.250', 'ff02::fa17'] as Array<Host>,
+    mdnsPort = 64023 as Port,
```

Comment:
Defaults here should come from the config if it's defined there. I think all the start parameters need to be fixed.

Reply:
This is no longer relevant due to MDNS being an injected optional dep